src/internal/observable/forkJoin.ts

```diff
@@ -13,9 +13,9 @@ export function forkJoin<T, T2, T3>(v1: ObservableInput<T>, v2: ObservableInput<
 export function forkJoin<T, T2, T3, T4>(v1: ObservableInput<T>, v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>): Observable<[T, T2, T3, T4]>;
 export function forkJoin<T, T2, T3, T4, T5>(v1: ObservableInput<T>, v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>, v5: ObservableInput<T5>): Observable<[T, T2, T3, T4, T5]>;
 export function forkJoin<T, T2, T3, T4, T5, T6>(v1: ObservableInput<T>, v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>, v5: ObservableInput<T5>, v6: ObservableInput<T6>): Observable<[T, T2, T3, T4, T5, T6]>;
-export function forkJoin<T>(v1: ObservableInput<T>): Observable<T[]>;
 export function forkJoin<T>(sources: Array<ObservableInput<T>>): Observable<T[]>;
 export function forkJoin<T>(...sources: Array<ObservableInput<T>>): Observable<T[]>;
+export function forkJoin<T>(v1: ObservableInput<T>): Observable<T[]>;
```

Comment:
not exactly sure, but prolly this could be go line 10 instead?

Reply:
Yeah, I agree.